common/decorators.py

```diff
@@ -70,10 +69,8 @@ def wrapper(request, *args, **kw):
             if isinstance(output, dict):
                 if request.is_ajax():
                     tpl = ('%s_body%s' % os.path.splitext(tpl), tpl)
-                return render_to_response(tpl,
-                                          output,
-                                          RequestContext(request),
-                                          content_type=ct)
+
+                return render(request, tpl, output, content_type=ct)
```

Comment:
good change. for future I'd also consider removing that decorator altogether, and instead handle the output format and template name in the view function/class directly. (for {read,debug}ability)

Reply:
I agree with removing the decorator - it is confusing in terms of program flow.